modules/overrides/runtime_config_overrides.go

```diff
@@ -241,6 +241,14 @@ func (o *runtimeConfigOverridesManager) WriteStatusRuntimeConfig(w io.Writer, r
 	return nil
 }
 
+func (o *runtimeConfigOverridesManager) GetLevel(_ Level) Interface {
+	return o
```

Comment:
Should it be an error to request a different level of overrides from this manager?  I.e. requesting `LevelUserConfigurable` from this will silently return the `LevelRuntime` overrides.   It looks like the [UserConfigurableOverridesManager does check](https://github.com/grafana/tempo/pull/3180/files#diff-d1960db0e556609f29af683322ba95e6cdd4da202f089cc08d14b3ecda5f4c1bR208).

Reply:
Yeah, that's a good point. It's kind of weird because the `runtimeConfigOverridesManager` will always be the lowest level, it doesn't have a child to call like the `UserConfigurableOverridesManager` does.

Maybe I should just keep it simple and call the method `GetRuntimeOverrides()` Interface. It's less generic but I'm not sure it's really worth introducing the concept of an overrides level if it isn't used anywhere else yet...